src/rust/engine/fs/src/store.rs

```diff
@@ -1937,6 +1943,51 @@ mod tests {
     Bytes::from(STR)
   }
 
+  pub fn big_file_fingerprint() -> Fingerprint {
+    Fingerprint::from_hex_string("8dfba0adc29389c63062a68d76b2309b9a2486f1ab610c4720beabbdc273301f")
+      .unwrap()
+  }
+
+  pub fn big_file_digest() -> Digest {
+    Digest(big_file_fingerprint(), big_file_bytes().len())
+  }
+
+  pub fn big_file_bytes() -> Bytes {
+    let mut f = File::open(
+      PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+        .join("testdata")
+        .join("all_the_henries"),
+    ).expect("Error opening all_the_henries");
+    let mut bytes = Vec::new();
+    f.read_to_end(&mut bytes)
+      .expect("Error reading all_the_henries");
+    Bytes::from(bytes)
+  }
+
+  pub fn extra_big_file_fingerprint() -> Fingerprint {
+    Fingerprint::from_hex_string("8ae6924fa104396614b99ce1f6aa3b4d85273ef158191b3784c6dbbdb47055cd")
+      .unwrap()
+  }
+
+  pub fn extra_big_file_digest() -> Digest {
+    Digest(extra_big_file_fingerprint(), extra_big_file_bytes().len())
+  }
+
+  pub fn extra_big_file_bytes() -> Bytes {
```

Comment:
Rather than re-implementing the file reading, how about just:
```
pub fn extra_big_file_bytes() -> Bytes {
  let mut bytes = big_file_bytes();
  bytes.extend(&big_file_bytes());
  bytes
}
```

(Part of me just wants to add more plays to all_the_henries so that it's over 1MB, though :P I'd wager making it all_the_histories would do the job...)